spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -1189,6 +1189,19 @@
                 :idv_in_person_usps_proofing_results_job_user_sent_to_fraud_review,
               )
             end
+
+            it 'does not deactivate the profile when the enrollment has expired' do
+              stub_request_expired_proofing_results
+
+              job.perform(Time.zone.now)
+              profile = pending_enrollment.reload.profile
+
+              expect(profile.fraud_review_pending_at).to be_nil
+              expect(profile.fraud_rejection_at).to be_nil
+              expect(job_analytics).not_to have_logged_event(
```

Comment:
I think @gina-yamada's comment supersedes mine, but if we're gonna keep this test I'd expect to see a specific expectation for the profile like `expect(profile).not_to be_active`

Reply:
☝️ That makes sense to me. I think in this case the profile was never actually active, so we're not testing a state transition, but it's still a reasonable assertion to have.